### ISSUE TITLE
Prepare for removing scripts and rename

### ISSUE THREAD

**Issue by bassosimone** [measurement-kit/ci-common#2]
This repository contains docker images as well as scripts for running CI, hence its name. Yet, most repositories have been converted to MKBuild, except a few.

This discussion mirrors https://github.com/measurement-kit/cmake-modules/issues/1 and, for all repositories listed there, the cure is to switch to MKBuild.

The exception is measurement-kit/measurement-kit, where we don't use any CMake module but we still have a dependency on this repository.

The proper fix, in this case, is to vendor the current version of the scripts as part of measurement-kit/measurement-kit. At that point, no other repository that we control is using scripts in here. So, we can proceed with renaming and making sure we only keep code to generate the docker images we need in here.

To summarize:

- [x] Implement https://github.com/measurement-kit/cmake-modules/issues/1 

- [x] vendor CI scripts in measurement-kit/measurement-kit

- [ ] remove CI scripts from here

- [ ] rename repository

- [ ] update README.md

**Comment by bassosimone** [comment]
We can proceed because the only consumer of the scripts, libndt, will soon stop using this repository as part of https://github.com/measurement-kit/libndt/pull/108, hence there is no need to keep around the scripts. Until libndt migrates it can just use the version of the scripts bound with the version of this repository it is pinned to, where this scripts are available.
